tools/windows/install-help/cal/customactiondata.cpp

```diff
@@ -34,24 +34,26 @@ bool CustomActionData::init(const std::wstring &data)
         return false;
     }
 
-    auto start = data.begin();
-    auto end = data.end();
-    std::wregex re(L"((\\w+)=(.+)?;\\s*\r?\n)");
-    std::match_results<decltype(start)> results;
-    while (std::regex_search(start, end, results, re))
+    std::wstringstream ss(data);
+    std::wstring token;
+
+    while (std::getline(ss, token))
     {
-        auto propertyValue = results[3].str();
-        propertyValue.erase(propertyValue.begin(), std::find_if(propertyValue.begin(), propertyValue.end(), [](int ch)
-        {
-            return !std::isspace(ch);
-        }));
-        if (propertyValue.length() > 0)
+        // now 'token'  has the key=val; do the same thing for the key=value
```

Comment:
```suggestion
        // now 'token' has the string "<key>=<value>" and we have to split the two
```
I found your original comment a bit confusing, I think this is better.